From: Andrew Morton <akpm@linux-foundation.org>
Subject: tools-mm-introduce-a-tool-to-assess-swap-entry-allocation-for-thp_swapout-fix
Date: Tue Jun 25 11:01:55 AM PDT 2024

tools/mm/thp_swap_allocator_test.c needs linux/mman.h, per Kairui Song

Cc: Barry Song <v-songbaohua@oppo.com>
Cc: Chris Li <chrisl@kernel.org>
Cc: David Hildenbrand <david@redhat.com>
Cc: "Huang, Ying" <ying.huang@intel.com>
Cc: Hugh Dickins <hughd@google.com>
Cc: Kairui Song <kasong@tencent.com>
Cc: Kalesh Singh <kaleshsingh@google.com>
Cc: Ryan Roberts <ryan.roberts@arm.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/mm/thp_swap_allocator_test.c |    1 +
 1 file changed, 1 insertion(+)

--- a/tools/mm/thp_swap_allocator_test.c~tools-mm-introduce-a-tool-to-assess-swap-entry-allocation-for-thp_swapout-fix
+++ a/tools/mm/thp_swap_allocator_test.c
@@ -27,6 +27,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <string.h>
+#include <linux/mman.h>
 #include <sys/mman.h>
 #include <errno.h>
 #include <time.h>
_
